api/lib/mno_enterprise/intercom_events_listener.rb

```diff
@@ -0,0 +1,79 @@
+require 'intercom'
+
+module MnoEnterprise
+
+  class IntercomEventsListener
+
+    attr_accessor :intercom
+
+    def initialize
+      if MnoEnterprise.intercom_api_key && MnoEnterprise.intercom_api_secret
+        self.intercom = ::Intercom::Client.new(app_id: MnoEnterprise.intercom_api_key, api_key: MnoEnterprise.intercom_api_secret)
+      else
+        self.intercom = nil
+      end
+
+    end
+
+    def info(key, current_user_id, description, metadata, object)
+      return unless self.intercom
+      u = User.find(current_user_id)
+      begin
```

Comment:
I don't think you need this `begin`, the `rescue Intercom::IntercomError` will apply to the whole method and it's easier to read.